frontend/src/ui/components/Header/Header.tsx

```diff
@@ -39,22 +67,63 @@ export const Header: React.FC = () => {
               </NavLink>
             </Route>
             <Route path={ROUTES.Console.HOME}>
-              <NavLink className="nav-link" to={ROUTES.Console.HOME}>
+              <NavLink
+                className="nav-link"
+                to={ROUTES.Console.HOME}
+                isActive={(match, location) => {
+                  // Make customize link active for any customization pages
```

Comment:
Typo: ```Mark this link as `active` for all customization pages.```

Let's not hardcode its name even in the comment and use backquotes to make the intentions clear.

Reply:
Thanks for the suggestion.